chromium/scripts/README

```diff
@@ -0,0 +1,28 @@
+These scripts help facilitate a build of PQC-enabled Chromium.
+They have been tested on Ubuntu 20.04 .
+
+How to use :
+
+1. Please set the PROJECT environment variable. This is the location where you
```

Comment:
If full path required in this variable (relative probably not sufficient?), please state it explicitly.

Reply:
Agrree. Fix upcoming.